paparazzi/src/main/java/app/cash/paparazzi/accessibility/AccessibilityRenderExtension.kt

```diff
@@ -63,8 +75,9 @@ public class AccessibilityRenderExtension : RenderExtension {
     }
   }
 
-  private var unmergedNodes: List<SemanticsNode>? = null
+  private var unmergedNodes: List<SemanticsNode> = mutableListOf()
```

Comment:
yep, this reference is a mem leak as the render extension lives around the same time as Paparazzi. Reverting this change and putting it inline to avoid the potential mem leak.